src/qp/quant_pdf.py

```diff
@@ -136,12 +144,63 @@ def locs(self):
         """Return the locations at which those quantiles are reached"""
         return self._locs
 
+    @property
+    def pdf_constructor_name(self):
```

Comment:
This and the following three new methods are the machinery that allow the user to dynamically select pdf constructors at run time.